packages/connectors/ledger-connector/src/iframe/provider.ts

```diff
@@ -0,0 +1,21 @@
+import {
+  IFrameEthereumProvider,
+  IFrameEthereumProviderOptions,
+} from '@ledgerhq/iframe-provider';
+
+interface RequestArguments {
+  readonly method: string;
+  readonly params?: any[];
+}
+
+export class LedgerIFrameProvider extends IFrameEthereumProvider {
+  constructor(options?: IFrameEthereumProviderOptions) {
```

Comment:
you don't need constructor if it stay the same

Reply:
fixed